pynetbox/core/api.py

```diff
@@ -213,19 +269,24 @@ def create_token(self, username, password):
 
     @contextlib.contextmanager
     def activate_branch(self, branch):
-        """
-        Context manager to activate the branch by setting the schema ID in the headers.
+        """Context manager to activate the branch by setting the schema ID in the headers.
```

Comment:
Might be worth noting here that this is a feature of netbox-branching and not available in core NetBox.